package.json

```diff
@@ -58,8 +58,9 @@
     "open": "^7.0.0"
   },
   "devDependencies": {
+    "@gmrchk/cli-testing-library": "^0.1.2",
     "jest": "^23.6.0",
-    "strip-ansi": "^6.0.0",
-    "standard": "^14.0.0"
+    "standard": "^14.0.0",
+    "strip-ansi": "^6.0.0"
```

Comment:
Query: What does changing the order of these 2 lines achieve?

Reply:
IIRC I used cli commands to install software, so it rearranged those automagically.